new-client/src/utils/FetchWrapper.js

```diff
@@ -57,6 +59,14 @@ class FetchWrapper {
     ).test(url);
   }
 
+  getMetaValue(key) {
+    const el = document.getElementsByName(key);
+    if (el && el[0]) {
+      return el[0]?.attributes?.content?.value;
+    }
+    return null;
+  }
+
```

Comment:
Is this really necessary, wouldn't something like this be more sufficient and work on all modern browsers anyway?
```javascript
document.querySelector("meta[name='hajk-client-use-cache-buster']").getAttribute("content");
```